doc/release-notes.md

```diff
@@ -0,0 +1,40 @@
+## Spring-base release notes
+
+### Version 2.0.0
+Modules that use spring-base library have to do some actions when upgrading to the 2.0.0 version.
+
+ 1. Update `spring-boot-starter-parent` version in pom.xml
+```xml
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.5.2</version>
+    <relativePath /> <!-- lookup parent from repository -->
+  </parent>
+```
+ 2. Update `folio-spring-base` version in pom.xml
+```xml
+    <dependency>
+      <groupId>org.folio</groupId>
+      <artifactId>folio-spring-base</artifactId>
+      <version>2.0.0</version>
+    </dependency>
+```
+ 3. `getUserName()` method is no longer supported in `FolioExecutionContext`. To retrieve username you could create feign client and use it to fetch needed user's data from `mod-users` by user's ID that is available in `FolioExecutionContext` by method `getUserId()`.
+```java
+@FeignClient(value = "users")
+public interface UsersClient {
+
+  @GetMapping(value = "/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+  Optional<UserDto> fetchUserById(@PathVariable("id") String id);
+
+  @Value
+  class UserDto {
+
+    String id;
+    String username;
+  }
+}
+```
+ 4. Default for Folio log4j2 configurations introduced: `log4j2.properties` for plain and `log4j2-json.properties` for json format. So, you could remove log4j2 configurations from your module. See details in [README](../README.md#default-logging-format).
+ 5. Default logging for incoming and outgoing requests introduced. You should decide if you will use this mechanism. See details in [README](../README.md#logging-for-incoming-and-outgoing-requests).
```

Comment:
could you change "if you will use" to "if you use" or rephrase "You should decide whether to use this mechanism or not"